target/linux/ipq40xx/patches-6.6/110-mtd-limit-OTP-nvmem-to-non-nand-devices.patch

```diff
@@ -47,12 +47,12 @@ Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
 
 --- a/drivers/mtd/mtdcore.c
 +++ b/drivers/mtd/mtdcore.c
-@@ -933,7 +933,7 @@ static struct nvmem_device *mtd_otp_nvme
- 	config.name = compatible;
- 	config.id = NVMEM_DEVID_AUTO;
```

Comment:
I think this patch can be removed as it's the same fix as https://github.com/openwrt/openwrt/commit/dd78a59cd7b029560b33cb3ac0e1aa8b747bd807.

Reply:
Thanks, fixed.

Reply:
My proposal combining the fix https://github.com/openwrt/openwrt/commit/dd78a59cd7b029560b33cb3ac0e1aa8b747bd807 and https://github.com/openwrt/openwrt/pull/14940/commits/8ec76705d42766156c4185f59998c7144f815363
```
-	config.add_legacy_fixed_of_cells = !!np;
+	config.add_legacy_fixed_of_cells = !!np && !of_node_name_eq(mtd->dev.of_node, "nand");

Reply:
My proposal combining the fix https://github.com/openwrt/openwrt/commit/dd78a59cd7b029560b33cb3ac0e1aa8b747bd807 and https://github.com/openwrt/openwrt/pull/14940/commits/8ec76705d42766156c4185f59998c7144f815363
```
-	config.add_legacy_fixed_of_cells = !!np;
+	config.add_legacy_fixed_of_cells = !!np && !of_node_name_eq(mtd->dev.of_node, "nand");